modules/kubernetes-addons/variables.tf

```
@@ -39,6 +39,21 @@ variable "irsa_iam_permissions_boundary" {
   description = "IAM permissions boundary for IRSA roles"
 }
 
+variable "eks_oidc_issuer_url" {
+  type        = string
+  description = "The OpenID Connect identity provider (issuer URL without leading `https://`)"
+}
+
+variable "aws_eks_cluster_endpoint" {
```

since this is net new, can we drop the preceding `aws_`
```suggestion
variable "eks_cluster_endpoint" {
```